src/installation/guides/chroot.md

```diff
@@ -0,0 +1,313 @@
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+### Install base-system (ROOTFS method only)
+
+> Note: This section can be skipped if following the XBPS method. You already
+> have base-system installed.
+
+ROOTFS images generally contain out of date software, due to being a snapshot of
+the time when they were built, and do not come with a complete `base-system`.
+Update the package manager and install `base-system`.
+
+```
+# xbps-install -Su xbps
+# xbps-install -u
+# xbps-install base-system
+# xbps-remove base-voidstrap
+```
+
+### Installation Configuration
+
+Write the hostname to `/etc/hostname`. Go through the options in
+[`/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+### Set a Root Password
+
+[Configure at least one super user account.](../../config/users-and-groups.md)
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+### Configure fstab
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX`, `/dev/nvmeXnYpZ`, etc. with their respective
+UUID, which can be found by found by running
+[blkid](https://man.voidlinux.org/blkid.8). Referring to filesystems by their
+UUID guarantees they will be found even if they are assigned a different name at
+a later time. In some situations, such as booting from USB, this is absolutely
+essential. In other situations, such as SATA or NVME, drives will usually always
+have the same name, unless drives are physically added or removed. Therefore,
+this step may not be strictly necessary but is almost always recommended.
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following fstab:
+
+```
+/dev/sda1       /boot/EFI   vfat    rw,relatime,[...]       0 0
+/dev/sda2       /           ext4    rw,relatime             0 0
+```
+
+The fstab is modified thus with information from blkid:
```

Comment:
"The information from \[blkid(8)\](https://man.voidlinux.org/blkid.8) results in the following \`/etc/fstab\`:"

Reply:
Okay. I did not make blkid a link though, as the rest of the document follows the convention of only linking to a manpage at the first reference.